HLTrigger/Muon/plugins/HLTMuonL1TFilter.cc

```diff
@@ -110,7 +118,15 @@ bool HLTMuonL1TFilter::hltFilter(edm::Event& iEvent,
       MuonRef muon(allMuons, distance(allMuons->begin(allMuons->getFirstBX()), it));
 
       // Only select muons that were selected in the previous level
-      if (find(prevMuons.begin(), prevMuons.end(), muon) == prevMuons.end())
+      bool matchPrevL1 = false;
```

Comment:
`deltaR2` is a rather expensive operation (ok, better than `deltaR`): I would suggest to move this check after all other, cheaper, ones